shared/text/api/v2.0/buckets/oss/create.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
This isn't used anywhere yet. Accidental commit. I'll be changing it to use client library.